### DISCUSSION TITLE
Force open 'Advanced' components-panel via block.json?

### DISCUSSION THREAD

**Discussion by IvanPr** [WordPress/gutenberg#49928] [Q&A]
Is there a way to force open the 'Advanced' components-panel section of the ACF Block.
Perhaps there could be a way to set initialOpen via block.json?

**Comment by skorasaurus** [comment]
That action is typically controlled through the initialOpen property in the [Panel ](https://developer.wordpress.org/block-editor/reference-guides/components/panel/)Component. 

Since ACF Panels' contents are typically completely through ACF, I would recommend to contact their support. 

If you'd like a real-life example of how it's done in non ACF blocks,
check https://gitlab.com/cpl/post-title-list-block/-/blob/53d20c77fb294d8bc0eec11a86069139e1665ce8/src/edit.js#L111 or https://github.com/WordPress/gutenberg/blob/88335dbf42f456b2c89e3fa12ade3d5170100daa/packages/block-library/src/social-link/edit.js#L101
